css/about.css

```diff
@@ -0,0 +1,698 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+:root {
+  --black: #272a31;
+  --orange: #ec5242;
+  --grey: #d3d3d3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato', sans-serif;
+  text-transform: capitalize;
+  text-decoration: none;
+  outline: none;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+html {
+  width: 100%;
+  scroll-behavior: smooth;
+}
+
+/* -----------------------------Hamburgar Icon Starts------------------------------------------------------ */
+.hamburger {
+  position: absolute;
+  cursor: pointer;
+  width: 20px;
+  height: 14px;
+  left: 14px;
+  top: 46px;
+  z-index: 99;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #fff;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+/* -----------------------------Hamburgar Icon Ends------------------------------------------------------ */
+
+/* -----------------------------Navbar Section Starts---------------------------------------------------- */
+
+#navbar {
+  position: fixed;
+  width: 100%;
+  left: -100%;
+  backdrop-filter: blur(5px);
+  background-color: #ec534285;
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] **[OPTIONAL]:** I noticed too that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓